server/src/main/java/org/elasticsearch/common/util/concurrent/TaskExecutionTimeTrackingEsThreadPoolExecutor.java

```diff
@@ -222,7 +258,17 @@ public Map<Runnable, Long> getOngoingTasks() {
     }
 
     // Used for testing
-    public double getEwmaAlpha() {
+    public double getExecutionEwmaAlpha() {
         return executionEWMA.getAlpha();
     }
+
+    // Used for testing
+    public double getQueueLatencyEwmaAlpha() {
+        return queueLatencyMillisEWMA.getAlpha();
+    }
+
+    // Used for testing
+    public boolean trackingQueueLatencyEwma() {
+        return trackQueueLatencyEWMA;
+    }
```

Comment:
Nit/Opinion: I would favour perhaps that we should not include "EWMA" in the name of these things, and just say "average" because I think how we calculate the average is an implementation detail?

Reply:
Sure. Changed most of the interfaces from EWMA to Average, except for `#getQueuedTaskLatencyMillisEWMA()`  that I changed to `#getQueuedTaskLatencyMillis()`: I figure whoever wants to know the details will have to go look.